### DISCUSSION TITLE
Mojolicious.org mojolicious version for docs.

### DISCUSSION THREAD

**Discussion by daleif** [mojolicious/mojo#2116] [General]
Is there any reason why mojolicious.org is still showing Mojolicious 9.32? For example the version of `Mojo::File` does not explain `spew` (https://docs.mojolicious.org/Mojo/File)

**Comment by jberger** [comment]
Something funny happened with the container image tags. It has been fixed for docs.mojolicious.org, I still need to work on docs.mojojs.org. Thanks for the report!
